src/domotica/domotica.h

```diff
@@ -29,12 +29,50 @@
 
 #include <stdint.h>
 #include <stdlib.h>
+#include "domotica_cv.h"
 #include "domotica_rx.h"
 
+
+// ----------------------------------------------------------------------------
+#ifndef DOMOTICA_OUTPUT_SIZE
+  #define DOMOTICA_OUTPUT_SIZE 16
+#endif
+
+#ifndef DOMOTICA_RX_INPUT_ADDRESS_SIZE
+  #define DOMOTICA_RX_INPUT_ADDRESS_SIZE 6
+#endif
+
+#ifndef DOMOTICA_FASTCLOCK_SIZE
+  #define DOMOTICA_FASTCLOCK_SIZE 60
+#endif
+
+// ----------------------------------------------------------------------------
+// Define the LNCV start and end ranges of the different elements, if not
+// specified in any other module.
+#ifndef DOMOTICA_LNCV_OUTPUT_BRIGHTNESS_START
+  #define DOMOTICA_LNCV_OUTPUT_BRIGHTNESS_START 14
+#endif
+#ifndef DOMOTICA_LNCV_OUTPUT_BRIGHTNESS_END
```

Comment:
Should these be overridable? I'd suggest to only make `DOMOTICA_OUTPUT_SIZE`, `DOMOTICA_RX_INPUT_ADDRESS_SIZE` and `DOMOTICA_FASTCLOCK_SIZE` overridable. Then these defines can be derived from those mentioned above and the checks can be made easier.